manage_breast_screening/notifications/management/commands/retry_failed_message_batch.py

```diff
@@ -0,0 +1,83 @@
+import json
+import os
+import time
+from logging import getLogger
+
+from django.core.management.base import BaseCommand, CommandError
+
+from manage_breast_screening.notifications.management.commands.command_helpers import (
+    MessageBatchHelpers,
+)
+from manage_breast_screening.notifications.models import (
+    MessageBatch,
+    MessageBatchStatusChoices,
+)
+from manage_breast_screening.notifications.services.api_client import ApiClient
+from manage_breast_screening.notifications.services.queue import Queue
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django Admin command which takes an ID of a MessageBatch with
+    a failed status and retries sending it to the Communications API.
+    """
+
+    def handle(self, *args, **options):
+        # get the batch from the queue in an iterator
+
+        queue = Queue.RetryMessageBatches()
+        failed_message_batch_iterator = queue.items(1)
+
+        # return if there isn't a message_batch
+        try:
+            failed_queue_message = failed_message_batch_iterator.next()
+        except StopIteration as e:
+            logger.info(e)
+            return
+
+        message_batch_id = json.loads(failed_queue_message.content)["message_batch_id"]
+        message_batch = MessageBatch.objects.filter(
+            id=message_batch_id, status="failed_recoverable"
```

Comment:
We now define all these statuses in models.py so probably better to refer to them with `MessageBatchStatusChoices.FAILED_RECOVERABLE.value` rather than a string.